avaframe/out3Plot/outAIMEC.py

```diff
@@ -386,7 +386,7 @@ def resultVisu(cfgPath, cfgFlags, rasterTransfo, resAnalysis, plim):
     # Get input data
     fnames = cfgPath['pressurefileList']
 
-    flag = cfgFlags['typeFlag']
+    flag = float(cfgFlags['typeFlag'])
```

Comment:
Why the float here? string or int should be fine, so no conversion is necessary, right?

Reply:
Ah, i forgot about the missing type conversion of the ini system